### ISSUE TITLE
kismatic 1.5.3 install error - unable to start Calico Network Components on KVM Centos7 machines

### ISSUE THREAD

**Issue by wymarsalis** [apprenda/kismatic#825]
Hi, trying to install K8s cluster on KVM machines and got the following error:

`Start Calico Network Components                                                 [ERROR]`
`- Task: label nodes with calico CNI provider  centos9:                  [ERROR]`                                                                         
`---- STDERR ----`
`The connection to the server 127.0.0.1:6443 was refused - did you specify the right host or port?`

On the master server, all the pods go down.
My configuration:
- 3 nodes with Centos7
- Master: 1 CPU, 6 G
- Etcd: 1 CPU, 2 G
- Worker: 1 CPU, 3 G
- SELinux disabled on all nodes
- kismatic executed from an Ubuntu 16.04 machine

Any idea? Why all the pods suddenly stop on the master?

**Comment by alexbrand** [comment]
Is there anything useful in the kubelet log (`journalctl -u kubelet`) or the API server logs? It's hard to tell without more information.

**Comment by wymarsalis** [comment]
Hi,
The problem has been solved, it was just a matter of increasing memory and disk. 
Worked with (master,etcd,worker)=(3,4,4) GB, 20GB for each. 
I also increased the retries number in almost all tasks playbooks. Some services doesn't startup within the expected time and cause the kismatic install procedure to abort, with no other solution to rollback other than snapshot-revert.
Nice and impressing job of automation done by Kismatic team.
